CreateJavaPackage.sh

```diff
@@ -24,7 +24,7 @@ esac
 cp /openapitools.json ./openapitools.json
 
 echo Generating API
-npx openapi-generator-cli generate -i ./spec.json -g java -o java_service --git-user-id $repoOwner --git-repo-id $repoId --global-property models,modelTests=false,modelDocs=false -p basePackage=mqube.caseService -p modelPackage=mqube.caseService.models -p dateLibrary=java8-localdatetime
+npx openapi-generator-cli generate -i ./spec.json -g java -o java_service --git-user-id $repoOwner --git-repo-id $repoId --global-property models,modelTests=false,modelDocs=false -p basePackage=$name -p modelPackage=${name}.models -p dateLibrary=java8-localdatetime
```

Comment:
@animit-kulkarni With the changes:
basePackage = Mqube.$SwaggerServiceName.Client 
modelPackage = Mqube.$SwaggerServiceName.Client.models

Is that okay or should modelPackage be Mqube.$SwaggerServiceName.models?